ads/medianet.md

```diff
@@ -44,19 +59,33 @@ The ad size depends on the ``width`` and ``height`` attributes specified in the
  * ``height`` 
  * ``type = "medianet"``
 
-For further configuration related details, please feel free to reach out to your Media.net contact.
+If you have questions, please feel free to reach out to your Media.net contact.
+
 
 ## Supported Parameters 
 
+###Media.net Contextual Monetization
+
+<strong>Mandatory Parameters</strong>
+ 
+* ``data-tagtype`` - This parameter represents the product the publisher is using; It should be <strong>``cm``</strong> for our <strong>Contextual Monetization solution</strong>
```

Comment:
why double backquote? Should be single backquote? 
In markdown, you can use `**text**` to make bold text, intead of using `<strong>`.